openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -18,45 +22,28 @@ class PersonalizedLearnerScheduleCallToAction:
 
     def get_ctas(self, xblock, category):
         """
-        Return the calls to action associated with the specified category for the given xblock.
-
-        See the CallToActionService class constants for a list of recognized categories.
-
-        Returns: list of dictionaries, describing the calls to action, with the following keys:
-                 link, link_name, form_values, and description.
-                 If the category is not recognized, an empty list is returned.
-
-        An example of a returned list:
-        [{
-            'link': 'localhost:18000/skip',  # A link to POST to when the Call To Action is taken
-            'link_name': 'Skip this Problem',  # The name of the action
-            'form_values': {  # Any parameters to include with the CTA
-                'foo': 'bar',
-            },
-            # A long-form description to be associated with the CTA
-            'description': "If you don't want to do this problem, just skip it!"
-        }]
+        Look at CallToActionService docstrings to see what will be returned.
```

Comment:
Maybe include the first line of the old docstring too, since that's the summary line and might at least give a general idea.